examples/express/README.md

```diff
@@ -0,0 +1,52 @@
+# Express.js example
+
+0. Installation
+
+    Navigate to the example subdirectory, for example:
+
+    ```bash
+    cd examples/express
+    ```
+
+    Then:
+
+    ```bash
+    npm install
+    ```
+
+0. Collect the openid-client [Issuer](https://github.com/panva/node-openid-client/tree/master/docs#issuer) metadata.
```

Comment:
This line is repeated.

Reply:
Which one, the "0. Collect . . ."? There are two for `Issuer` and `Client`.

Reply:
Oh, right. I guess they looked so similar I didn't notice the different links. It would be worth saying a little more for the "Issuer" line. It's not clear to me exactly what metadata needs to be collected.

Reply:
Well, then I'd be repeating a lot of things already available at the link location and described in the main README. If OIDC configuration doc is accessible, the single .well-known URI is all they may need.

Reply:
Maybe just say something about using the well-known endpoint, then.

Reply:
But I'll add a comment in that regard.

Reply:
Did.